### ISSUE TITLE
Add support for AMM LpTokens

### ISSUE THREAD

**Issue by jtfirek** [interlay/interbtc-squid#65]
Once support for MMA tokens are merged into the lib type currencyExt, we should add support on the squid side. Here is a comment by Peter about the matter.
https://github.com/interlay/interbtc-squid/pull/62#discussion_r1088834377
I have added the need functionally into a commented section in the code for #62, but this code needs to be uncommented once the support for these tokens is live in the lib.

**Comment by bvotteler** [comment]
A lib version supporting those currencies and the method mentioned by Peter has been added in the latest squid version.
Meaning: This issue is now unblocked.

**Comment by bvotteler** [comment]
PR #82 completed the work needed for this issue.
